src/main/java/pippi/parser/DateFormatter.java

```diff
@@ -0,0 +1,41 @@
+package pippi.parser;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class DateFormatter {
+    public static final DateTimeFormatter format = DateTimeFormatter.ofPattern("yyyy-MM-dd");

```

Comment:
Since this is static and final, it should be considered a constant right?
Should you capitalize it?
![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/107026005/e7564b65-1bf6-4aba-96a4-91091b5e2438)